README.rst

```diff
@@ -233,9 +234,22 @@ Saving Keystrokes By Setting The Primary Domain
 
 To save some keystrokes, you can set ``primary_domain = 'js'`` in conf.py and then say (for example) ``autofunction`` rather than ``js:autofunction``.
 
+Typescript support
+------------------
+
+There is experimental typescript support in sphinx-js.  You can enable this by setting the sphinx variable ``json_generator = 'typedoc'``.
+You will not need to install jsdoc, but you do need to install typedoc::
+
+  npm install -g typedoc
+
+Version 0.11.1 is known to work.
+
 Configuration Reference
 -----------------------
 
+``json_generator``
```

Comment:
``js_parser``? ``js_documentation_extractor``? I feel like we should say it's "JS" (because I can imagine other things within the Sphinx pipeline needing to deal with JSON in some scenario) and de-emphasize JSON (since it's an implementation detail that users shouldn't need to think about).

Reply:
good point

Reply:
How about `js_language` being either `javascript` (default) or `typescript`?